source/convective_grad/_parameters

```diff
@@ -4,3 +4,5 @@ small_temp     real         -1.e200
 small_dens     real         -1.e200
 
 plotfile       string       ""
+
+spherical 	   bool         false
```

Comment:
we've been moving away from using bool in the parameters scripts.  Can you just make this `int`